### ISSUE TITLE
Baseline indicators should provide percentile dropdown rather than textbox

### ISSUE THREAD

**Issue by fungjj92** [azavea/climate-change-lab#317]
<img width="667" alt="screen shot 2017-11-10 at 12 31 00 pm" src="https://user-images.githubusercontent.com/10568752/32670835-0aba67fc-c613-11e7-8c72-1a1d8fd92e7e.png">

The API only calculates ClimateDataBaseline for certain percentiles (1, 5, 95, 99). Reusing the percentile component which provides a text box right now (percentile indicators should indeed have a text box) is inappropriate.  Instead it should be a dropdown of these 4 options.

**Comment by flibbertigibbet** [comment]
Related to/duplicate of #243

**Comment by flibbertigibbet** [comment]
See also comments [here](https://github.com/azavea/climate-change-components/blob/f4dbd09fc7400d344c540c19e6095865a92897b1/src/lib/modules/charts/components/extra-params/percentile.component.ts#L50) and [here](https://github.com/azavea/climate-change-components/blob/a9d1f36eb8d54cd7ebe0777f24c2b04dd7b104f8/src/lib/modules/charts/components/extra-params/percentile-historic.component.ts#L60) in what is now the components repository.

**Comment by rmartz** [comment]
This is specific to the `Extreme __ Events` indicators, which are different from the other percentile indicators in that they're limited to choosing only from a defined set of legal options (Which, as an aside, [it appears we don't mention in our documentation](https://docs.climate.azavea.com/indicators.html#extreme-precipitation-events)). The `Percentile __` indicators use any numeric percentile and can keep their free-form input, so we'll still need validation for them but as a distinct task from this.

**Comment by rmartz** [comment]
I believe this is blocked by azavea/climate-change-api#757

**Comment by fungjj92** [comment]
Moved to azavea/climate-change-components#14
